.storybook/addons.js

```diff
@@ -1,6 +1,3 @@
 import '@storybook/addon-options/register';
 import '@storybook/addon-links/register';
-
-if (process.env.NODE_ENV === 'documentation') {
```

Comment:
couldn't find any situations where we were in a docs context without being in a documentation node env.  Check seems unnecessary.